CMake/SlicerExtensionCPack.cmake

```diff
@@ -59,6 +59,14 @@ endforeach()
 include(SlicerMacroExtractRepositoryInfo)
 SlicerMacroExtractRepositoryInfo(VAR_PREFIX ${EXTENSION_NAME})
 
+#-----------------------------------------------------------------------------
+# Set extension revision
+#-----------------------------------------------------------------------------
+if(NOT "${${EXTENSION_NAME}_FORCED_REVISION}" STREQUAL "")
```

Comment:
The variable should be `"${${EXTENSION_NAME}_FORCED_WC_REVISION}"` (`WC_` was missing).